docs/vc-http-api.yml

```diff
@@ -89,6 +89,30 @@ paths:
           description: invalid input!
         "500":
           description: error!
+  /credentials/verifyMany:
```

Comment:
This is an anti-pattern that raises a number of architectural concerns:

1. Why do you need this, why can't you scale horizontally? We're able to handle 1,000s of verifications per second w/o this API... what scaling issues are you having?
2. This is a DoS attack vector that requires implementations to put DoS mitigations in their application code, rather than protecting themselves at the edges of a network.

To be clear -- feel free to define this as an optional API and implement it for your Phase I... but I expect there to be strong push back in a future WG that attempts to standardize this. You will note that Twitter, Facebook, Github, Pinterest, etc. don't provide batch APIs for their most heavily utilized endpoints -- there is a reason for this.

Reply:
issue discussed at length in #88 welcome comments on that issue.

Reply:
This argument feels stronger for verification than it does for issuance.

factories, tend to make more than 1 thing at a time.

Reply:
> factories, tend to make more than 1 thing at a time.

Yes, and they scale via parallelization (more assembly lines), not by making the one machine in the middle of the factory floor bigger... but the analogy is strained... we're talking about computing architecture here, which does things in ways that aren't done on factory floors.

If folks want to insist that we need batch APIs, let's insist that they demonstrate that keeping simple/atomic APIs that can be parallelized don't work. 

Moving discussion to #88.